### PR TITLE
sicar url changed

### PR DESCRIPTION
**Pull request by rupestre-campos**
As SICAR download url have changed, data fails to download.
Following changes address to fix urls in code in order to reflect govmnt updates.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,7 +3,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+*.venv
```
Comment (urbanogilson) [inline .gitignore:6]: Let's remove the specifics of the environment

```suggestion
```

**Review by urbanogilson — CHANGES_REQUESTED** [review]
Thanks for the PR! It looks like it failed the linter check because it's not compliant with Black. Could you please format the code using Black and push the changes?

File: .gitignore (hunk 1 above)
Comment (rupestre-campos) [reply .gitignore]: removed in new commit

Comment (rupestre-campos) [reply .gitignore]: new commit with fix for black formatting and also could run the tests locally to catch another minor issues. 
When installing libs for testing I got numpy 2.0 version with a older version of open-cv which did not end well so I added up also a version upper limit for numpy and open-cv proven to work.

**Review by urbanogilson — APPROVED** [review]
Thanks for the PR

**Pull request merged**